### ISSUE TITLE
cp: cannot stat '../../dist/traffic_ops-8.0.0-0..el8.x86_64.rpm': No such file or directory

### ISSUE THREAD

**Issue by zrhoffman** [apache/trafficcontrol#7921]
## This Bug Report affects these Traffic Control components:

- CDN in a Box

## What did you do?

1. Download Apache Traffic Control 8.0.0 release candiate 5 <[https://dist.apache.org/repos/dist/dev/trafficcontrol/8.0.0/RC5/apache-trafficcontrol-8.0.0.tar.gz](https://dist.apache.org/repos/dist/dev/trafficcontrol/8.0.0/RC5/apache-trafficcontrol-8.0.0.tar.gz)> and ungzip it
2. Navigate to the CDN in a Box directory
3. Try to build the Traffic Ops RPM:
   ```shell
   make --debug traffic_ops/traffic_ops.rpm
   ```

## What did you expect to happen?

Traffic Ops RPM is built and placed at `infrastructure/cdn-in-a-box/traffic_ops/traffic_ops.rpm`

## What actually happened?

The RPM builds successfully but is not copied:
```
[...]
Results in 'dist':
total 32676
-rw-r--r-- 1 user user    89774 Jan 22 16:42 build-traffic_ops.log
-rw-r--r-- 1 user user 17229352 Jan 22 16:42 traffic_ops-8.0.0-12732.0c515199.el8.src.rpm
-rw-r--r-- 1 user user 16130264 Jan 22 16:42 traffic_ops-8.0.0-12732.0c515199.el8.x86_64.rpm
cp -f "../../dist/traffic_ops-8.0.0-0..el8.x86_64.rpm" "traffic_ops/traffic_ops.rpm"
cp: cannot stat '../../dist/traffic_ops-8.0.0-0..el8.x86_64.rpm': No such file or directory
make: *** [Makefile:169: traffic_ops/traffic_ops.rpm] Error 1
```

The RPM being built is `traffic_ops-8.0.0-12732.0c515199.el8.x86_64.rpm`, but the RPM the Makefile expects is `traffic_ops-8.0.0-0..el8.x86_64.rpm` (with a missing commit hash).

The Makefile should detect that we are not in a git repository and instead get the build number from `BUILD_NUMBER`.

**Comment by ocket8888** [comment]
Does this only affect the debug RPMs?

**Comment by zrhoffman** [comment]
> Does this only affect the debug RPMs?

No, `--debug` is unnecessary to reproduce (and is a `make` flag, not one of ours)
